.gitignore

```diff
@@ -1,4 +1,5 @@
 __pycache__
+*.mo
```

Comment:
@PetrDlouhy, this adds `.mo` files to gitignore, but also includes `.mo` file. That's suspicious.